packages/mantine/src/components/collection/Collection.tsx

```diff
@@ -19,19 +19,85 @@ import {CollectionItem} from './CollectionItem';
 import useStyles from './Colllection.styles';
 
 interface CollectionProps<T> extends DefaultProps<Selectors<typeof useStyles>> {
+    /**
+     * The default value each new item should have
+     */
     newItem: T;
+    /**
+     * A render function called for each item passed in the `value` prop.
+     *
+     * @param item The current item's value
+     * @param index The current item's index
+     */
     children: (item: T, index: number) => ReactNode;
-    defaultValue?: T[];
+    /**
+     * The list of items to display inside the collection
+     *
+     * @default []
+     */
     value?: T[];
+    /**
+     * The initial items of the collection (for uncontrolled usage only)
+     */
+    defaultValue?: T[];
+    /**
+     * Unused, has no effect
+     */
     onFocus?: () => void;
```

Comment:
I think I initially added it because getInputProps would provide it and forgot to remove it. I just didn't want to mix its removal with the adding of jsdocs